net/natmap/Makefile

```diff
@@ -0,0 +1,38 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=natmap
+PKG_VERSION:=20221111
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/heiher/natmap.git
+PKG_SOURCE_DATE:=2022-11-11
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_MIRROR_HASH:=4359bdddd16d7f97af4ee03f814ea4e649554c30eeed93e5f4fc586151d2476e
```

Comment:
is it possible to download source code via git codeload?

Reply:
oh i see that, thanks for clarification.

Reply:
Yup, that's a good idea - proposed the same https://github.com/heiher/natmap/issues/6